docs/DeveloperGuide.adoc

```diff
@@ -303,6 +303,24 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* *` |insurance agent prone to mistakes with many entries to manually key in |be sure that the
+details of the buyers are valid without having to manually check every individual record|speed up my
+workflow without having to worry that I might have accidentally keyed in the details wrongly
+or if I have collected invalid information from buyers.
+
+|`* *` |insurance agent with many clients to keep track of |be sure that each individual in my records is only in
+possession of policies he or she is qualified for|
+
+|`* *` |insurance agent with many clients and policies to keep track of |get suggestions of appropriate policies when I assign an inappropriate policy to a buyer
+|quickly rectify the error
+
+|`* *` |insurance agent with many details to key in |get suggestions of auto-corrected commands when I make a typo |speed up my workflow
+possession of policies|
```

Comment:
Extra bar at the end which caused formatting error
![Screenshot 2019-09-22 at 7 34 00 PM](https://user-images.githubusercontent.com/40198156/65386561-f8fdc600-dd6f-11e9-98e1-3cc2c247ac44.png)